client/static/leaflet/ruler/leaflet.ruler.js

```diff
@@ -54,6 +54,7 @@
       this._clickedLatLong = null
       this._clickedPoints = []
       this._totalLength = 0
```

Comment:
we can drop this next debug line, right @lebaphi ?

Reply:
Stupid GH comments 🤣 

I was referring to the console.log on line 57

Reply:
ah I see 🤣🤣